### DISCUSSION TITLE
I believe there's a typo in code for DidReceiveRemoteNotification in beta-0124

### DISCUSSION THREAD

**Discussion by imsam67** [shinyorg/shiny#1255] [Q&A]
I believe I noticed a typo in beta-0124. Attaching a screenshot. In `AppDelegate` on line for `public void DidReceiveRemoteNotification()`, the method name in Shiny library is `OnDidReceiveRemoveNotification()` -- see attached image.

![Screenshot 2023-06-08 145232](https://github.com/shinyorg/shiny/assets/15844030/3c8b4c60-0db0-4bd9-9e39-6dcda3b22f43)

**Comment by aritchie** [comment]
Ahh indeed.  Thanks for reporting this.
